packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/index.tsx

```diff
@@ -43,12 +43,12 @@ const DividerMiddle = styled.div`
     display: flex;
     align-items: center;
     padding: 5px 20px;
-    color: ${colors.NEUE_TYPE_LIGHT_GREY};
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
     font-weight: ${variables.FONT_WEIGHT.MEDIUM};
     font-size: ${variables.FONT_SIZE.SMALL};
     border-radius: 25px;
-    border: 1px solid ${colors.NEUE_STROKE_GREY};
-    background: ${colors.WHITE};
+    border: 1px solid ${props => props.theme.STROKE_GREY};
+    color: ${props => props.theme.BG_WHITE};
```

Comment:
original 'background', new: 'color', color is doubled, is that expected?

Reply:
good catch :) I'll probably ping invity team to take a look at their part of the app as they are more likely to notice unusual things